# BEST NEIGHBORHOOD

## Introduction

<font size="4">To find the best neighborhood, we used datasets provided by WPRDC. The metrics being used are </font>
* <font size = "4">Cleanliness
* <font size = "4">Arrest Counts
* <font size = "4">Average Air Pollution
  <br>
<font size ="4">
These three metrics impact the quality of life and community well-being of the neighborhood. Cleanliness serves as an indicator of how well-maintained and cared for a neighborhood is. It shows the community pride and services, which can influence property values. Arrest counts provide a look into public safety and crime levels in the neighborhoods. Neighborhoods with fewer arrests typically offer residents a greater sense of safety and a better environment for families and individuals. Average air pollution measures environmental health, which could affect an individual's health. Together, these metrics allow us to see a neighborhood's livability. 
</font>

In [1]:
import pandas as pd
import matplotlib.pyplot as plt
import fpsnippets

## Cleanliness

## Arrest Counts

## Average Air Pollution

<font size="4">To determine the best neighborhood in Pittsburgh, we analyzed average air pollution levels across different neighborhoods using data from the Allegheny County Air Quality Emissions dataset. This is a metric since it can impact a resident's health.</font>

### Steps

<font size="4">We calculated the average air pollution for each neighborhood by:</font>
1. <font size ="4">Grouping emissions by their geographic coordinates
2. <font size ="4">Sorting those geographic coordinates into their respective neighborhood
3. <font size ="4">Finding the average tons of pollutants per year for each neighborhood
4. <font size ="4">Identifying the lowest neighborhoods

### Results

In [4]:
df = pd.read_csv('AQEData.csv')

df['neighborhood'] = df.apply(
    lambda row: fpsnippets.geo_to_neighborhood(row['lat'], row['lon']),
    axis=1
)

neighborhood_stats = df.groupby('neighborhood').agg({
    'tons_per_yr': 'mean',  
    'lat': 'count'  
}).reset_index()

neighborhood_stats.columns = ['neighborhood', 'avg_tons_per_yr', 'num_reports']

neighborhood_stats_sorted = neighborhood_stats.sort_values(by='avg_tons_per_yr', ascending=True)

print("Neighborhoods with the lowest average air pollution:")
print(neighborhood_stats_sorted.head(15))

Neighborhoods with the lowest average air pollution:
                neighborhood  avg_tons_per_yr  num_reports
5                    Carrick         0.101935           31
14             Homewood West         0.101935           31
11                    Esplen         0.224615           26
24       Squirrel Hill South         0.246129           31
20        Point Breeze North         0.911000          130
27       Upper Lawrenceville         2.206758          171
15                   Larimer         3.340062          160
17       Lower Lawrenceville        17.921686          162
12                 Fairywood        19.017500           16
23          South Side Flats        39.216818           66
3                  Brookline        40.503105          233
4       California-Kirkbride        90.072273           22
2                      Bluff        95.582119          397
8                    Chateau       148.523571           14
16  Lincoln-Lemington-Belmar       150.384040           99


<font size = "4">What we can see is that our lowest average is Carrick at 0.101935 tons per year.</font>

## THE BEST NEIGHBORHOOD

<font size="4">The best neighborhood between all three metrics has been different, but within our metrics, there is one neighborhood that stands out as best between the three. That neighborhood is **Esplen.** Esplen is placed eleventh for cleanliness, tenth for lowest arrest counts, and third for lowest average air pollution. This is a better result than any other neighborhood from our three metrics.</font>